docs/en/ingest-management/elastic-agent/elastic-agent-standalone-logging.asciidoc

```diff
@@ -0,0 +1,126 @@
+
+The Logging section of the `elastic-agent.yml` config file contains settings for configuring the logging output.
+The logging system can write logs to the `syslog`, `file`, `stderr`, `eventlog` or rotate log files.
+If you do not explicitly configure logging, the `stderr` output is used.
+
+["source","yaml",subs="attributes"]
+----
+agent.logging.level: info
+agent.logging.to_files: true
+agent.logging.files:
+  path: /var/log/elastic-agent
+  name: elastic-agent
+  keepfiles: 7
+  permissions: 0600
+----
+
+[discrete]
+[[elastic-agent-standalone-logging-settings]]
+== Logging configuration settings
+
+You can specify the following settings in the Logging section of the `elastic-agent.yml` config file.
+
+[cols="2*<a"]
+|===
+| *Setting* | *Description*
+| `agent.logging.level` | The minimum log level. 
+
+Possible values: 
+
+* `error`: Logs errors and critical errors.
+
+* `warning`: Logs warnings, errors, and critical errors.
+
+* `info`: Logs informational messages, including the number of events that are published. Also logs any warnings,
+errors, or critical errors.
+
+* `debug`: Logs debug messages, including a detailed printout of all events flushed. Also logs informational messages,
+warnings, errors, and critical errors. When the log level is `debug`, you can specify a list of *selectors* to display
+debug messages for specific components. If no selectors are specified, the `*` selector is used to display debug
+messages for all components.
+
+Default: `info`
+
+| `agent.logging.selectors` | Specify the selector tags that are used by different {agent} components for debugging.
+To debug the output for all components, use `*`. To display debug messages related to event
+publishing, set to `publish`. Multiple selectors can be chained.
+
+Possible values: `[beat]`, `[publish]`, `[service]`
+
+| `agent.logging.to_stderr` | Set to `true` to write all logging output to the `stderr` output—this is equivalent to
+using the `-e` command line option.
+
+Default: `true`
+
+| `agent.logging.to_syslog` | Set to `true` to write all logging output to the `syslog` output.
+
+Default: `false`
+
+| `agent.logging.to_eventlog` | Set to `true` to write all logging output to the Windows `eventlog` output.
+
+Default: `false`
+
+| `agent.logging.metrics.enabled` | Set to `true` for {agent} to periodically log its internal metrics that have changed in the
+last period. For each metric that changed, the delta from the value at the beginning of the period is logged.
+Also, the total values for all non-zero internal metrics get logged on shutdown.
+
+Default: `true`
+
+| `agent.logging.metrics.period` | Specify the period after which to log the internal metrics.
+
+Default: `30s`
+
+| `agent.logging.to_files` | Set to `true` to log to rotating files. Set to `false` to disable logging to files.
+
+Default: `true`
+
+| `agent.logging.files.path` | The directory that log files is written to.
+
+include::{tab-widgets}/logging-widget.asciidoc[]
```

Comment:
cool...wasn't sure if a widget would work inside of a table cell. It would be interesting to see what the experience is like someone using a screen reader. (Probably not worth your time to test this right now because at some point we'll do some accessibility testing that will flag issues.)

Reply:
Very neat